server.js

```diff
@@ -219,40 +262,24 @@ app.use('/', require('./controllers/archived'));
  * - Apply section specific controller logic
  * - Add common routing (for static/fully-CMS powered pages)
  */
-forEach(routes.sections, function(section, sectionId) {
+forEach(routes, function(section, sectionId) {
     const router = express.Router();
 
     /**
      * Add section locals
      * Used for determining top-level section for navigation and breadcrumbs
      */
     router.use(function(req, res, next) {
-        const locale = req.i18n.getLocale();
-        res.locals.sectionId = sectionId;
         res.locals.sectionTitle = req.i18n.__(`global.nav.${sectionId}`);
-        res.locals.sectionUrl = localify(locale)(req.baseUrl);
+        res.locals.sectionUrl = localify(req.i18n.getLocale())(req.baseUrl);
         next();
     });
 
     /**
-     * Page-level logic
-     * Apply page level middleware and mount router if we have one
+     * Mount page router
      */
     section.pages.forEach(function(page) {
-        router
-            .route(page.path)
-            .all(
-                injectCopy(page.lang),
-                injectHeroImage(page.heroSlug),
-                (req, res, next) => {
```

Comment:
This was my favourite middleware to remove

```
(req, res, next) => {
    next();
}
```

That's right it just calls `next` 🤷‍♂

Reply:
wow.

Reply:
thank u, next